packages/plugins/documentation/server/services/documentation.js

```diff
@@ -119,6 +119,10 @@ module.exports = ({ strapi }) => {
         await fs.ensureFile(apiDocPath);
         const apiPathsObject = builApiEndpointPath(api);
 
+        if (!apiPathsObject) {
+          continue;
+        }
+
```

Comment:
I propose to move `await fs.ensureFile(apiDocPath);` after the if, otherwise it creates an empty file

Reply:
sure